src/tic_tac_toe/player.clj

```diff
@@ -0,0 +1,23 @@
+(ns tic-tac-toe.player)
+
+(def human-player "X")
+(def computer-player "O")
+
+(defn create-player
+  [token]
+  token)
+
+(defn provide-move
+  [number]
+  number)
+
```

Comment:
is provide-move or create-player used?? I'm not sure why they're here

Reply:
deleted